Flagsmith-engine/Feature/Models/FeatureStateModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Flagsmith_engine.Utils;
+using System.Linq;
+
+namespace Flagsmith_engine.Feature.Models
+{
+    public class FeatureStateModel
+    {
+        [JsonProperty(PropertyName = "feature")]
+        public FeatureModel Feature { get; set; }
+        [JsonProperty(PropertyName = "enabled")]
+        public bool Enabled { get; set; }
+        [JsonProperty("feature_state_value")]
+        public object Value { get; set; }
+        [JsonProperty(PropertyName = "multivariate_feature_state_values")]
+        public List<MultivariateFeatureStateValueModel> MultivariateFeatureStateValues { get; set; }
+        [JsonProperty(PropertyName = "django_id")]
+        public int DjangoId { get; set; }
+        public string FeatureStateUUID { get; set; }
+        public object GetValue(int? identityId = null) =>
+            identityId.HasValue && MultivariateFeatureStateValues.Count > 0 ? GetMultivariateValue(identityId.Value) : Value;
+
+        public object GetMultivariateValue(int identityId)
```

Comment:
@asadabbasfolio3 looks like this has just been updated to string. Can we change the input argument to `var` ? Or perhaps add an override function so we have something like: 

```dotnet
public object GetMultivariateValue(int identityId) { return GetMultivariateValue(identityId.ToString()); }

public object GetMultivariateValue(string identityId) {...}
```

Reply:
the function calling inside this function i.e `get_hashed_percentage_for_object_ids()` is implicitly converting input to string to create a hash.so we found no need to write its overload for int as well because it is converting it into string in the end.

Reply:
Yep, ok, fair enough.